WcaOnRails/config/locales/en.yml

```diff
@@ -363,7 +363,7 @@ en:
         organizer_ids: "Competition organizers (they need an account to be listed here)."
         external_website: "The website of the competition. Must be a valid http(s) url."
         registration_open: "Note: Registration open and close are in UTC time"
-        remarks: "Some additional information you want to pass on to the board. For example, reasons why this competition was requested less than one month away, if applicable. Please fill this out in English!"
+        remarks: "Some additional information you want to pass on to the board. For example, elaborate your reasons if this competition is requested less than four weeks away, or if you are requesting the application of Article Z (https://www.worldcubeassociation.org/regulations/#article-Z-optional). Please fill this out in English!"
```

Comment:
Could you include a screenshot of this? Is the user going to have to copy paste this link in order to view it? Could we make this an actual html link somehow?